app/uk/gov/hmrc/bindingtariffclassification/search/Search.scala

```diff
@@ -20,18 +20,18 @@ import java.time.Instant
 
 import play.api.mvc.QueryStringBindable
 import uk.gov.hmrc.bindingtariffclassification.model.CaseStatus
-import uk.gov.hmrc.bindingtariffclassification.model.CaseStatus.CaseStatus
-import uk.gov.hmrc.bindingtariffclassification.sort.SortDirection.SortDirection
-import uk.gov.hmrc.bindingtariffclassification.sort.SortField.SortField
+import uk.gov.hmrc.bindingtariffclassification.model.CaseStatus._
+import uk.gov.hmrc.bindingtariffclassification.sort.SortDirection._
 import uk.gov.hmrc.bindingtariffclassification.sort.{SortDirection, SortField}
+import uk.gov.hmrc.bindingtariffclassification.sort.SortField._
 
 import scala.util.Try
 
 
 case class Search
 (
   filter: Filter = Filter(),
-  sort: Option[Sort] = None
+  sort: Option[Sort] = Some(Sort(field = SortField.COMMODITY_CODE, direction = SortDirection.ASCENDING))
```

Comment:
Let me know if you agree here.
I think we want to sort by commodity code in case nothing has been specified. 
But if you disagree, I could remove this change and let the front-end or service layer to decide the sorting.

Reply:
I’d disagree with this, the backend should “do as it is told” and we want different sorts in different contexts so it’s better for the ui to specify sort field and direction for all requests (which it does)

Reply:
The logic before was essentially “if you ask for sort, your results are sorted, otherwise they are not”
And 
“If you ask for sort but don’t specify a direction, we’ll default it”
Which I think was fine from a “what would I expect to happen” point of view